src/main/java/org/fundacion/jala/converter/view/compiler/CodeTextArea.java

```diff
@@ -54,6 +56,7 @@ public String getText() {
 
             /**
              * Adds the number of rows in Code Area Text.
+             *
```

Comment:
Why the constructor has methods, maybe you forgot key

Reply:
It will be reviewed when CodeTextArea is refactored.